plugins/modules/lvol.py

```diff
@@ -41,13 +41,13 @@
     description:
     - The size of the logical volume, according to lvcreate(8) --size, by
       default in megabytes or optionally with one of [bBsSkKmMgGtTpPeE] units; or
-      according to lvcreate(8) --extents as a percentage of [VG|PVS|FREE];
+      according to lvcreate(8) --extents as a percentage of [VG|PVS|FREE|ORIGIN];
       Float values must begin with a digit.
     - When resizing, apart from specifying an absolute size you may, according to
       lvextend(8)|lvreduce(8) C(--size), specify the amount to extend the logical volume with
       the prefix V(+) or the amount to reduce the logical volume by with prefix V(-).
     - Resizing using V(+) or V(-) was not supported prior to community.general 3.0.0.
-    - Please note that when using V(+) or V(-), the module is B(not idempotent).
+    - Please note that when using V(+), V(-), or percentage of sizes, the module is B(not idempotent).
```

Comment:
Shouldn't it be idempotent for at least some of the percentages? Like percentage of VG or PVS?

Reply:
Idempotence does not mean that if you use a percentage with respect to another quantity, that the object must not change even if that other quantity changes.

Reply:
Fair enough, let me update this to just %FREE. Thanks!